CMakeLists.txt

```diff
@@ -80,6 +80,8 @@ if (NETWORKIT_MONOLITH)
 			CXX_STANDARD ${NETWORKIT_CXX_STANDARD}
 			COMPILE_FLAGS "${NETWORKIT_CXX_FLAGS}"
 			LINK_FLAGS "${NETWORKIT_LINK_FLAGS}")
+	install(TARGETS networkit
+			LIBRARY DESTINATION .)
```

Comment:
We do not want to install the library in `/usr/local` here. In `setup.py` we set the installation prefix so that the library is installed into the Python package directory. For manual installations of the C++ library, we probably do want to install to `/usr/local/lib`.

Reply:
Okay, so `CMAKE_INSTALL_PREFIX` points to `/usr/local/` at UNIX systems. To let users use the C++ lib without need to specify a install prefix, we should change the code like some kind of this to make the `cmake ..` command possible on an linux machine.

```
if(NETWORKIT_PYTHON)
...
    install(TARGETS _Networkit networkit LIBRARY DESTINATION .)
...
else()
    install(TARGETS networkit LIBRARY DESTINATION lib)
endif()
```